benchmark/kernels/fused_moe_triton/benchmark_pre_reorder_triton.py

```diff
@@ -0,0 +1,85 @@
+import argparse
+import torch
+import triton
+import triton.language as tl
+
+from sglang.srt.layers.moe.ep_moe.kernels import pre_reorder_triton_kernel
+
+
+@triton.testing.perf_report(
+    triton.testing.Benchmark(
+        x_names=["batch_size"],
+        x_vals=[64, 128, 256, 512, 640, 768, 1024],
```

Comment:
The batch size (bs) and topk can be combined using the product function. Refer to https://github.com/sgl-project/sglang/blob/main/sgl-kernel/benchmark/bench_moe_align_block_size.py#L248 for more details.

Reply:
Revised to use product.